frontend/apps/erd-web/next.config.ts

```diff
@@ -1,7 +1,9 @@
 import type { NextConfig } from 'next'
 
 const nextConfig: NextConfig = {
-  /* config options here */
+  outputFileTracingIncludes: {
+    '/erd/p/\\[\\.\\.\\.slug\\]': ['./prism.wasm'],
+  },
```

Comment:
Define the file to be referenced from the request handler function corresponding to the path on Vercel.


ref: [How can I use files in Vercel Functions?](https://vercel.com/guides/how-can-i-use-files-in-serverless-functions)